docs/src/main/paradox/typed/interaction-patterns.md

```diff
@@ -112,7 +113,12 @@ Java
 
 ## Adapted Response
 
-Most often the sending actor does not, and should not, support receiving the response messages of another actor. In such cases we need to provide an @apidoc[actor.typed.ActorRef] of the right type and adapt the response message to a type that the sending actor can handle.
+Most often the sending actor does not, and should not, support receiving the response messages of another actor.
+
+In such cases we need to provide an @apidoc[actor.typed.ActorRef] of the right type and adapt the response message
+to a type that the sending actor can handle. In the case of Scala, we need to make a distinction between Scala 2
+and Scala 3. In the latter case, we can actually get rid of the need to adapt the response message which vastly
+simplifies the handling of responses. The details can be found in the section @ref:[Handling actor responses in Scala 3](handling-actor-responses-scala3.md).
```

Comment:
By leveraging the `Union type`?

Reply:
Good point!
Forgot the add this to my last PR update.
Will push another update.